### PR TITLE
PdfPreview Widget Enhancements

### PR DESCRIPTION
**Pull request by marcos930807**
Added pdfPreviewPageDecoration, scrollViewDecoration properties to PdfPreview Widget

### CODE REVIEW

File: printing/lib/src/pdf_preview.dart [hunk 1]
```diff
@@ -13,20 +13,22 @@ import 'raster.dart';
...
+      this.allowSharing = true,
+      this.maxPageWidth,
+      this.canChangePageFormat = true,
+      this.actions,
+      this.pageFormats,
+      this.onError,
+      this.onPrinted,
+      this.onShared,
+      this.scrollViewDecoration,
+      this.pdfPreviewPageDecoration})
```
Comment (DavBfr) [inline printing/lib/src/pdf_preview.dart:30]: can you add the last comma? Before } to keep the alignment.

Comment (marcos930807) [reply printing/lib/src/pdf_preview.dart]: Done

**Comment by DavBfr** [comment]
Merged.

**Comment by cassioseffrin** [comment]
@DavBfr , could you tell me if there is some feature request to make PdfPreview support zoom through pinch gesture, or any way to make it possible? For big sizes like A4, A3 it's very important.

**Comment by DavBfr** [comment]
Yes, #422 asks for it. If you have an idea on how to do it, you can create a PR.

**Comment by cassioseffrin** [comment]
I will take a closer look on the source code to see if I can help. Thks @DavBfr

**Pull request closed without merging**